Gemfile

```diff
@@ -4,6 +4,3 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in cocina-models.gemspec
 gemspec
-
-gem 'byebug'
-gem 'rspec_junit_formatter' # For CircleCI
```

Comment:
I don't know why we need to declare dev/test dependencies in two places, so I moved this to the gemspec.